packages/editor/src/components/visual-editor/style.scss

```diff
@@ -3,6 +3,8 @@
 	height: 100%;
 	display: block;
 	background-color: $gray-300;
+	// Make this a stacking context to contain the z-index of children elements.
+	isolation: isolate;
```

Comment:
😬 I just learned about this

Reply:
Same 😅

Reply:
> 😬 I just learned about this

And yet you [used the property here](https://github.com/WordPress/gutenberg/pull/62676/commits/2212dbfefaff02c937156cd09db904676c512e49#diff-5065a00f747fc9cdddcc2220e19e8e56df99e52f99cc25e1a4e51a21daf0897aR41) 😆

Reply:
I know I know, that's where I saw it first. I copied it form the View Transition Guide (by Google) to reset cross-shadow transitions :P